src/runtime/module_importer.js

```diff
@@ -13,6 +13,7 @@ exports.initTracer = function () {
     const path = require.resolve("dd-trace", { paths: ["/var/task/node_modules", ...module.paths] });
     // tslint:disable-next-line:no-var-requires
     const tracer = require(path).init({
+        debug: true,
```

Comment:
I am assuming this enables debug logging for the tracer? Is that what we want?

Reply:
Ooh good eye! Need to remove this, thanks!